src/ducks/transfers/TransferPage.jsx

```diff
@@ -370,9 +422,14 @@ class TransferPage extends React.Component {
       accounts.data
     )
 
+    const { isMobile } = this.props.breakpoints
+    const PersonalInfoComponent = isMobile
```

Comment:
Slightly surprised by this, Modals on mobile are fullscreen so I'm not sure what the difference between a page and a not closable, full screen modal is.

Reply:
The fullscreen modal would hide the bottom navigation tabs.